README.md

```diff
@@ -65,22 +65,53 @@ all project repositories at once.
 
 ## Quick Start
 
-The easiest way to try the demo is to use [provided Docker images](https://hub.docker.com/orgs/helidon/sockshop/repositories)
+The easiest way to try the demo is to use [provided Docker images](https://hub.docker.com/orgs/helidonsockshop/repositories)
 and Kubernetes deployment scripts from this repo. 
 
 Kubernetes scripts depend on Kustomize, so make sure that you have a newer version of `kubectl`
-that supports it. If you do, you can simply run
+that supports it (at least 1.14 or above). 
+
+If you do, you can simply run the following command from the `sockshop` directory
+and replace `BACKEND` with one of the following indicating the type of back-end you want to deploy.
+
+* core - Core back-end
+* mongo - MongoDB back-end
+* mysql - MySQL database back-end 
+* redis - Redis back-end
+
+```bash
+$ kubectl apply -k k8s/BACKEND 
```

Comment:
Each backend has to be in a separate namespace, so maybe we should add `-n BACKEND` to all `kubectl` commands that need the namespace to be specified.